### ISSUE TITLE
Installing dvclive breaks dvc's version command

### ISSUE THREAD

**Issue by gusmith** [iterative/dvclive#63]
I wanted to test dvclive in a project I'm already using dvc.
My current version of dvc is 2.0.5.
After install dvclive, the command `dvc --version` returns `0.0.1` (which is thus also wrongly shown when running `dvc repro ...`). 

Step to reproduce with python 3.8.7 on a Mac:
```
pip install dvc==2.0.5
pip install dvclive
dvc --version
```

Some information (after installing `dvc` and `dvclive`:
```
$pip list
Package           Version
----------------- ---------
appdirs           1.4.4
atpublic          2.1.3
cached-property   1.5.2
certifi           2020.12.5
cffi              1.14.5
chardet           4.0.0
colorama          0.4.4
commonmark        0.9.1
configobj         5.0.6
decorator         4.4.2
dictdiffer        0.8.1
diskcache         5.2.1
distro            1.5.0
dpath             2.0.1
dulwich           0.20.20
dvc               2.0.5
dvclive           0.0.1
flatten-dict      0.3.0
flufl.lock        3.2
fsspec            0.8.7
ftfy              5.9
funcy             1.15
future            0.18.2
gitdb             4.0.5
GitPython         3.1.14
grandalf          0.6
idna              2.10
jsonpath-ng       1.5.2
mailchecker       4.0.3
nanotime          0.5.2
networkx          2.5
packaging         20.9
pathlib2          2.3.5
pathspec          0.8.1
phonenumbers      8.12.19
pip               21.0.1
ply               3.11
psutil            5.8.0
pyasn1            0.4.8
pycparser         2.20
pydot             1.4.2
pygit2            1.5.0
Pygments          2.8.1
pygtrie           2.3.2
pyparsing         2.4.7
python-benedict   0.23.2
python-dateutil   2.8.1
python-fsutil     0.4.0
python-slugify    4.0.1
PyYAML            5.4.1
requests          2.25.1
rich              9.13.0
ruamel.yaml       0.16.13
ruamel.yaml.clib  0.2.2
setuptools        49.2.1
shortuuid         1.0.1
shtab             1.3.5
six               1.15.0
smmap             3.0.5
tabulate          0.8.9
text-unidecode    1.3
toml              0.10.2
tqdm              4.59.0
typing-extensions 3.7.4.3
urllib3           1.26.4
voluptuous        0.12.1
wcwidth           0.2.5
xmltodict         0.12.0
zc.lockfile       2.0
```

It was tested in a project and repeated in a fresh virtual environment with the same output (i.e. `dvc --version` returning `0.0.1`).

But surprisingly, if installing first `dvclive` and then `dvc`, `dvc --version` returns the expected `2.0.5`.

**Comment by discdiver** [comment]
Installing dvclive and running `dvc version` shows 0.0.1 for me, too.  Additionally, running `dvc exp run` , stopping training, and then running `dvc exp run` again results in the following output: 

```
+-----------------------------------------+
|                                         |
|     Update available 0.0.1 -> 2.0.6     |
|     Run `pip install dvc --upgrade`     |
|                                         |
+-----------------------------------------+
```

**Comment by pared** [comment]
@gusmith @discdiver `0.0.2` Should have the bug fixed
